name-registry/tests/functions/expiry.rs

```diff
@@ -0,0 +1,34 @@
+mod passing {
+    use crate::utils::*;
+
+    #[tokio::test]
+    async fn can_get_expiry() {
+        let (instance, _id, _wallet) = get_contract_instance().await;
+
+        let name = String::from("SwaySway");
+
+        register(&instance, &name, 5000).await;
+
+        let old_expiry = expiry(&instance, &name).await;
+
+        extend(&instance, &name, 5000).await;
+
+        let new_expiry = expiry(&instance, &name).await;
+
+        assert_eq!(old_expiry + 5000, new_expiry);
+    }
+}
+
+mod failing {
+    use crate::utils::*;
+
+    #[tokio::test]
+    #[should_panic]
```

Comment:
```suggestion
    #[should_panic(expected = "Revert(42)")]
```

Could you add this to all tests to make sure they are panicking in the way you intend? We've caught some bugs by retroactively adding this before.

Reply:
Thank you, that found some flaws in my tests